packages/rest/test/unit/rest.server/rest.server.open-api-spec.unit.ts

```diff
@@ -89,6 +89,29 @@ describe('RestServer.getApiSpec()', () => {
     });
   });
 
+  it('ignores routes marked as "x-internal" via app.route(route)', () => {
+    function greet() {}
+    function meet() {}
+    server.route(
+      new Route(
```

Comment:
Let's use the recently introduced sugar API instead of calling `new Route`.

```ts
server.route('get', '/greet', {'x-internal': true, /*...*/}, greet);
```

Reply:
VSC is complaining `No overload expects 4 arguments, but overloads do exist that expect either 1 or 6 arguments.`.

Did you mean something like this?

```
server.route('get', '/greet', spec, MyController, factory, 'greet');
```

We'll have to create `MyController` and `factory`, in that case. It adds more to the test code.